src/utils/helpers/Checkout.ts

```diff
@@ -0,0 +1,43 @@
+import { FilterOptions, OrderStatus, PaymentMethod, PaymentStatus, SortOptions } from '@/utils'
+
+export const getOrderStatusByEnum = (orderEnum: OrderStatus) => {
+  return orderEnum === OrderStatus.Delivered
+    ? 'Đã giao hàng'
+    : orderEnum === OrderStatus.Delivering
+      ? 'Đang giao hàng'
+      : orderEnum === OrderStatus.Confirmed
+        ? 'Đã xác nhận'
+        : orderEnum === OrderStatus.Pending
+          ? 'Đang xử lí'
+          : 'Đã hủy'
+}
```

Comment:
tạo ra 1 cái obj message key là OrderStatus.Delivered: value là 'Đã giao hàng'
làm kiểu giống vầy
`import { HttpStatus } from '@nestjs/common';

export const exceptionCase = {
  [ErrorCode.EMAIL_ALREADY_REGISTERED]: {
    status: HttpStatus.CONFLICT,
    message: 'Email is already registered!',
    errorCode: ErrorCode.EMAIL_ALREADY_REGISTERED,
  },
  [ErrorCode.EMAIL_NO_AUTHENTICATED]: {
    status: HttpStatus.CONFLICT,
    message: 'Email has not been authenticated!',
    errorCode: ErrorCode.EMAIL_NO_AUTHENTICATED,
  },`